packages/circuit-ui/components/NotificationFullscreen/NotificationFullscreen.stories.tsx

```diff
@@ -0,0 +1,49 @@
.526V89.375Z' fill='black'/%3E%3Cpath d='M174.8 98.3242H166.835C165.174 98.3242 163.828 99.6706 163.828 101.331C163.828 102.992 165.174 104.339 166.835 104.339H174.8C176.461 104.339 177.807 102.992 177.807 101.331C177.807 99.6706 176.461 98.3242 174.8 98.3242Z' fill='white'/%3E%3Cpath d='M198.172 75.6289H165.274C164.584 75.6289 163.902 75.7694 163.269 76.0417C162.635 76.314 162.064 76.7125 161.589 77.2128L150.065 89.3736H188.614L199.255 78.1456C199.456 77.934 199.59 77.6682 199.641 77.3812C199.692 77.0942 199.657 76.7985 199.542 76.5307C199.427 76.263 199.236 76.0348 198.992 75.8746C198.749 75.7143 198.464 75.6289 198.172 75.6289Z' fill='%238FBBFF'/%3E%3Cpath d='M131.459 112.661H79.7397C77.8101 112.661 76.6789 110.479 77.7827 108.892L91.3695 89.375H150.065L135.23 110.688C134.807 111.297 134.243 111.794 133.587 112.137C132.93 112.481 132.2 112.66 131.459 112.661Z' fill='%23BEDCFF'/%3E%3Cpath d='M143.355 50.8897C144.381 50.8064 145.406 50.7066 146.43 50.5904C148.77 50.3265 151.105 49.9841 153.413 49.5354C155.4 49.1627 157.364 48.6788 159.296 48.0859L159.298 48.0852C161.218 47.4634 163.086 46.6934 164.886 45.7822C167.358 44.5569 169.872 43.1125 171.843 41.1216C172.758 40.1981 173.612 39.0908 173.873 37.7524L173.873 37.7524C174.154 36.3092 173.804 34.8545 173.21 33.5615L173.21 33.5614C171.869 30.648 169.381 28.4731 166.765 26.7652C163.981 24.9433 160.907 23.6061 157.753 22.5927L157.753 22.5927C154.566 21.5718 151.29 20.8729 148.009 20.287L148.009 20.287C146.408 20.0018 144.804 19.743 143.201 19.4845L143.201 19.4844V19.4844L143.199 19.4841C142.926 19.4434 142.741 19.6408 142.687 19.8307C142.66 19.9279 142.661 20.0415 142.713 20.1414C142.768 20.2468 142.87 20.3169 143.003 20.3358C143.003 20.3358 143.002 20.3357 143.002 20.3356L143.025 20.1875C146.224 20.692 149.424 21.2181 152.583 21.9409C155.742 22.6636 158.84 23.5807 161.791 24.8888C163.264 25.5411 164.688 26.2999 166.05 27.1593C167.356 27.9868 168.626 28.9026 169.731 29.9887L139.811 65.0575M143.355 50.8897C143.95 51.4749 144.526 52.0815 145.079 52.7085L145.079 52.709C146.095 53.8683 147.102 55.3115 147.644 56.8545C148.187 58.4001 148.268 60.0561 147.414 61.6212C146.717 62.901 145.547 63.8953 144.299 64.582C142.926 65.3467 141.385 65.758 139.813 65.7789L139.812 65.7789C139.677 65.7798 139.566 65.7263 139.49 65.6385C139.418 65.5538 139.385 65.446 139.385 65.344C139.386 65.2419 139.418 65.1342 139.491 65.0496C139.565 64.962 139.676 64.9077 139.81 64.9075M143.355 50.8897C143.289 50.8951 143.223 50.9004 143.157 50.9056C143.109 50.8591 143.061 50.8127 143.013 50.7664M139.81 64.9075C139.811 64.9075 139.811 64.9075 139.811 64.9075V65.0575M139.81 64.9075C139.81 64.9075 139.81 64.9075 139.809 64.9075L139.811 65.0575M139.81 64.9075C141.244 64.8888 142.651 64.5103 143.9 63.8066L143.901 63.8063C145.08 63.1495 146.159 62.2129 146.753 61.0029L146.888 61.0689C146.276 62.3151 145.17 63.2713 143.974 63.9373C142.702 64.6535 141.271 65.0387 139.811 65.0575' fill='black' stroke='black' stroke-width='0.3'/%3E%3C/svg%3E",
```

Comment:
Since the main use case will likely be SVG assets, maybe this component should accept a JSX element instead (for example an SVG imported in userland via svgr, so for example:

```ts
import { ReactComponent as MySvg } from "./mysvg.svg";

const Userland = () => (
  <NotificationFullscreen
    imageSrc={MySvg}
    // other props
  />
)
```

If we prefer this I'd change the prop name to simply `image` and make it accept either a `JSX.Element` or a `string` (to also support passing image urls).

Reply:
SVGs can also be passed in as URLs 😉 I like the idea of supporting both.